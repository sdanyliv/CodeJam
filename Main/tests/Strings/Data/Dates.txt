2000-1-10
2000-1-2
1999-12-25
2000-3-23
1999-3-3
2000-03-24
2000-03-21