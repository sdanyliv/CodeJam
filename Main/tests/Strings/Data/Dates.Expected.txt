1999-3-3
1999-12-25
2000-1-2
2000-1-10
2000-03-21
2000-3-23
2000-03-24